pkg/karmadactl/util/util.go

```diff
@@ -0,0 +1,27 @@
+package util
```

Comment:
Could we set a more accurate name? Like `pkg/karmadactl/util/deleteconfirmation.go`